logs_build/EventHubs/function.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Since we call the folder EventHubs_Metrics, shouldn't we call this EventHubs_Logs?

Reply:
done